public/styles/allArticles.scss

```diff
@@ -0,0 +1,94 @@
+.main {
+
+  #primary {
+
+    .category-cont {
+      overflow: hidden;
```

Comment:
Overflow here should be visible as well. This is making the cards get cut on the hover effect

Reply:
I have fixed this.